src/expr.rs

```diff
@@ -831,13 +831,7 @@ fn rewrite_string_lit(context: &RewriteContext,
     if context.config.format_strings == false {
         return Some(context.snippet(span));
     }
-    // Check if there is anything to fix: we always try to fixup multi-line
-    // strings, or if the string is too long for the line.
-    let l_loc = context.codemap.lookup_char_pos(span.lo);
-    let r_loc = context.codemap.lookup_char_pos(span.hi);
-    if l_loc.line == r_loc.line && r_loc.col.to_usize() <= context.config.max_width {
-        return Some(context.snippet(span));
-    }
+
     let fmt = StringFormat {
```

Comment:
iirc, string formatting can be expensive, so skipping it might be a win. OTOH, checking a multi-line string is within bounds is still O(n), so this might be premature optimisation.